lms/envs/common.py

```diff
@@ -2638,6 +2638,14 @@
 NOTIFICATION_EMAIL_CSS = "templates/credit_notifications/credit_notification.css"
 NOTIFICATION_EMAIL_EDX_LOGO = "templates/credit_notifications/edx-logo-header.png"
 
+
+################################ Settings for Microsites ################################
+
```

Comment:
It would be good to provide a list of parameters which can be specified in this location